docs/source/index.rst

```diff
@@ -155,6 +155,7 @@ Documentation
 
    cloud-setup/cloud-permissions/index
    cloud-setup/cloud-auth
+   cloud-setup/cloud-prepare
```

Comment:
Instead of having it in the root level, how about we place it under `cloud-setup/cloud-permissions/vsphere.rst` where we have the setup for other clouds as well?